src/metabase/api/public.clj

```diff
@@ -531,6 +553,44 @@
      :export-format :api
      :parameters    parameters :qp-runner qp.pivot/run-pivot-query)))
 
+(def ^:private action-execution-throttle
+  "Rate limit at 1 action per second on a per action basis.
+   The goal of rate limiting should be to prevent very obvious abuse, but it should
+   be relatively lax so we don't annoy legitimate users."
+  (throttle/make-throttler :action-uuid :attempts-threshold 1 :initial-delay-ms 1000 :delay-exponent 1))
+
+#_{:clj-kondo/ignore [:deprecated-var]}
+(api/defendpoint-schema POST "/action/:uuid/execute"
+  "Execute the Action.
+
+   `parameters` should be the mapped dashboard parameters with values.
+   `extra_parameters` should be the extra, user entered parameter values."
+  [uuid :as {{:keys [parameters], :as _body} :body}]
+  {uuid        su/UUIDString
+   parameters  (s/maybe {s/Keyword s/Any})}
```

Comment:
Why not make this a Malli schema too?

Reply:
Sorry, that sounded passive-aggressive. Since you did the work to define `UUIDString` twice I assume there's a good reason and so I'm curious

Reply:
I was probably just being lazy. I've updated it to use Malli now, thanks